extensions/default/src/ViewerLayout/index.jsx

```diff
@@ -183,7 +183,7 @@ function ViewerLayout({
       </Header>
       <div
         className="flex flex-row flex-nowrap items-stretch w-full overflow-hidden"
-        style={{ height: 'calc(100vh - 52px' }}
+        style={{ height: 'calc(100vh - 116px' }}
```

Comment:
should this be a fixed number of pixels or a percentage? Will this work on every screen?

Reply:
@legend620 it should be dynamic and should follow the header height of our nl-pacs. we could pass the height through attritbutes from the nl-pacs or use "flex" to fill up the container